### PR TITLE
Switch to cht-conf

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

medic/cht-core#6586

# Code review items

- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- Tested: Unit and/or integration tests where appropriate
- Backwards compatible: Works with existing data and configuration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
hey @garethbowen . What do you think?

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-Medic Project Configurer
+CHT Project Configurer
```
Comment (garethbowen) [inline README.md:1]: While you're in there, I think this would be more on brand:

```suggestion
CHT App Configurer
```

File: src/bin/cht-conf.js [hunk 2]
```diff
@@ -3,12 +3,12 @@
 const { error } = require('../lib/log');
```
Comment (garethbowen) [inline src/bin/cht-conf.js:3]: Maybe this file should just be called `index.js` or something?

**Review by garethbowen — APPROVED** [review]
Very thorough! Thanks.

Two small suggestions inline, but approved so you're not blocked.

File: src/bin/cht-conf.js (hunk 2 above)
Comment (dianabarsan) [reply src/bin/cht-conf.js]: Done!

**Pull request merged**